Lib/test/test_ctypes/test_dlerror.py

```diff
@@ -119,5 +123,17 @@ def test_null_dlsym(self):
             self.assertEqual(os.read(pipe_r, 2), b'OK')
 
 
+class TestCAPI(unittest.TestCase):
+
+    @unittest.skipUnless(hasattr(_ctypes, 'dlopen'), 'require ctypes.dlopen()')
+    @test.support.run_with_locales('LC_ALL', 'fr_FR.utf8', 'fr_FR.iso88591')
+    def test_localized_error(self):
+        with self.assertRaisesRegex(
+            OSError,
+            re.escape("foo.so: Ne peut ouvrir le fichier d'objet partagé"),
```

Comment:
Yes, and since I don't need to create it, I don't encounter issues with such filename. Depending on whether the locale is ISO-88591 or not, `\xff` decodes differently so it's fine (note that the OSError being raised does not have the error message as bytes but a string, so I can't check against bytes directly (I think?)).